manifests/init.pp

```diff
@@ -185,6 +185,7 @@
 # @param rabbitmq_group OS dependent, default defined in param.pp. The system group the rabbitmq daemon runs as.
 # @param rabbitmq_home OS dependent. default defined in param.pp. The home directory of the rabbitmq deamon.
 # @param $rabbitmqadmin_package OS dependent. default defined in param.pp. If undef: install rabbitmqadmin via archive, otherwise via package
+# @param $mqa_download_opts. default defined in params.pp.  Extra options to pass to Archive resource to download rabbitmqadmin
```

Comment:
Maybe `archive_options` would be more clear? The strings docs will add the default, so, even while this text is in some of the other lines, that bit is not actually needed.

Reply:
This needs to be updated too (to match the other places).